promslog/slog_test.go

```diff
@@ -94,6 +95,41 @@ func getLogEntryLevelCounts(s string, re *regexp.Regexp) map[string]int {
 	return counters
 }
 
+func TestDurationValues(t *testing.T) {
+	var (
+		buf    bytes.Buffer
+		output string
+		dur, _ = time.ParseDuration("1m30s")
+		config = &Config{Writer: &buf}
+	)
+
+	// Test logfmt.
+	logger := New(config)
+	logger.Info("duration testing", slog.Duration("duration_raw", dur), slog.String("duration_string", dur.String()))
```

Comment:
couldn't we just `logger.Info("duration testing", "duration_raw", dur, "duration_string", dur.String())`? as that's how it'd be usually used.

Reply:
We can indeed and that would be equivalent. Considering this is a test suite and we're testing behavior for specific types, it seemed appropriate to explicitly add these values in a type-deterministic way rather than letting slog's sugar logger calls do the type conversion stuff behind the scenes. 

Opinions?

Reply:
I'd go with testing as much of the logic as possible under conditions that closely mimic real‑world usage.